0.Preprocess_Data/scripts/3.data_aggregation.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pathlib
+
+import numpy as np
+import pandas as pd
+
+# In[2]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+aggregation = True
+nomic = True
+
+
+# In[ ]:
+
+
+if aggregation == True & nomic == True:
```

Comment:
Consider using solely the True boolean value of the variables or the `not` keyword to help with these if checks. Also, I recommend using the `and` keyword instead of `&` unless performing an explicit bitwise AND.

For example:
```suggestion
if aggregation and nomic:
```